### PR TITLE
abstract model names `<Noun> -> Model`

### PR DESCRIPTION
**Pull request by stephenlacy**

### CODE REVIEW

File: server/resources/user/deleteById.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import User from './model'
+import Model from './model'
 import { screenDeep } from 'palisade'
 
-export const isAuthorized = ({ id, user }) =>
-  User.authorized('delete', user, { id })
+export const isAuthorized = ({ id, model }) =>
```
Comment (yocontra) [inline server/resources/user/deleteById.js]: user is actually the session user object in isAuthorized

Comment (stephenlacy) [inline server/resources/user/deleteById.js]: ah - willfix

**Comment by stephenlacy** [comment]
updated

File: server/resources/user/model.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import rethink from 'connections/rethink'
 import palisade from 'palisade'
 const { type } = rethink
 
-const User = rethink.createModel('User', {
+const Model = rethink.createModel('Model', {
```
Comment (yocontra) [inline server/resources/user/model.js]: string argument should still be the model name

Comment (yocontra) [inline server/resources/user/model.js]: maybe get this from the folder name?

**Comment by stephenlacy** [comment]
@contra updated, it now gets the current directory and uppercases the first letter

Let me know if you want me to use es6 for the fn style

**Comment by yocontra** [comment]
Might be over-engineered, I don't think its a big deal to write the model name

**Comment by stephenlacy** [comment]
updated

**Pull request merged**